README.md

```diff
@@ -1,3 +1,3 @@
 # FaceLogin
-Thanks to [@legoboy0215](http://github.com/legoboy0215) for most of the code. You can download a compiled .phar file by clicking [here](https://github.com/Muqsit/FaceLogin/releases/download/v1.0/FaceLogin_v1.0.0.phar).
+Thanks to [@legoboy0215](http://github.com/legoboy0215) for most of the code. You can download a compiled .phar file by clicking [here](https://github.com/Amitminer/FaceLogin/releases/download/stable/FaceLogin_v2.0.0.phar).
```

Comment:
"You can download a compiled .phar" part can completely be removed. This only exists because the plugin came before Poggit.

Reply:
ok!